src/ResourceManager/ServiceFabric/Commands.ServiceFabric/Commands/UpdateAzureRmServiceFabricDurability.cs

```diff
@@ -0,0 +1,166 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Linq;
+using System.Management.Automation;
+using System.Threading.Tasks;
+using Microsoft.Azure.Commands.ServiceFabric.Common;
+using Microsoft.Azure.Commands.ServiceFabric.Models;
+using Microsoft.Azure.Management.Compute;
+using Microsoft.Azure.Management.Compute.Models;
+using Microsoft.Azure.Management.ServiceFabric.Models;
+using Newtonsoft.Json.Linq;
+using ServiceFabricProperties = Microsoft.Azure.Commands.ServiceFabric.Properties;
+
+namespace Microsoft.Azure.Commands.ServiceFabric.Commands
+{
+    [Cmdlet(VerbsData.Update, CmdletNoun.AzureRmServiceFabricDurability), OutputType(typeof(PSCluster))]
+    public class UpdateAzureRmServiceFabricDurability : ServiceFabricClusterCmdlet
+    {
+        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true,
+                   HelpMessage = "Specify durability Level")]
+        [ValidateNotNullOrEmpty()]
+        [Alias("DurabilityLevel")]
+        public DurabilityLevel Level { get; set; }
+
+        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true,
+                   HelpMessage = "Specify Service Fabric node type name")]
+        [ValidateNotNullOrEmpty()]
+        public string NodeTypeName { get; set; }
```

Comment:
NodeType - this is general advice.  Adding 'Name' at the end of most entities adds little value